packages/utilities/psammead-test-helpers/package.json

```diff
@@ -27,7 +27,8 @@
     "snapshot"
   ],
   "dependencies": {
-    "jest-styled-components": "^6.3.3"
+    "jest-styled-components": "^7.0.0",
+    "styled-components": "^5.0.0"
```

Comment:
It looks like a new dependency on `styled-components` has been added here. Please can you confirm why this has been added, when `psammead-test-helpers` doesn't use it directly?